problems/strings.js

```diff
@@ -103,24 +171,51 @@ function disemvowel() {}
  * @returns {string}
  */
 
- function disemvowelLongWords(){}
+//function disemvowelLongWords() {}
+const disemvowelLongWords = (string) => {
+//   let newStr = string. split(" ");
+//  let modified = ""; // how do i iterate through the newStr???
+//  let word = "";
+//   for (let i = 0; i < newStr.length; i++) {
+//    word = newStr[i];
+//     if (???.length >= 5) {
+//       modified += .replace(/[aeiou]/gi, "");
+//     } else {
+//       modified += ???; 
+//     }
+//   }
+//   return newStr;
+};
 
 /**
  * Takes in a string and returns the middle char.
  * If there is no character in the middle return the middle two characters
  * @param {string}
  * @returns {string}
  */
-function middleCharacter() {}
-
+//function middleCharacter() {}
+const middleCharacter = (string) => {
+  if(string.length % 2 === 1) {
+    return string[Math.floor(string.length/2)]
+  } else {
+    return string[Math.floor(string.length/2) - 1] + string[Math.floor(string.length/2)]  
+  }
+}
 /**
  * Takes in a string and reverses it.
  * Do not use split
  * @param {string}
  * @returns {string}
  */
 
-function reverseString() {}
+//function reverseString() {}
+const reverseString = (string) => {
+  let newStr = "";
+  for (let i = string.length - 1; i >= 0; i--) {
+    newStr += string[i];
```

Comment:
The reverses the entire sentence, the question wants the word order to remain the same, but the spelling to be reverses